<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IsingModelWidget</class>
 <widget class="QMainWindow" name="IsingModelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="modelTab">
       <attribute name="title">
        <string>Model</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>221</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="displayLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>221</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uSeriesTab">
       <attribute name="title">
        <string>U series</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="PlotView" name="uSeriesView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uHistTab">
       <attribute name="title">
        <string>U</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="PlotView" name="uHistView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="uVsTTab">
       <attribute name="title">
        <string>U vs T</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PlotView" name="uVsTView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cVsTTab">
       <attribute name="title">
        <string>CvVsT</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="PlotView" name="cVsTView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mSeriesTab">
       <attribute name="title">
        <string>M series</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="PlotView" name="mSeriesView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mHistTab">
       <attribute name="title">
        <string>M</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="PlotView" name="mHistView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mVsTTab">
       <attribute name="title">
        <string>M vs T </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="PlotView" name="mVsTView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="xVsTTab">
       <attribute name="title">
        <string>ChiVsT</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="PlotView" name="xVsTView" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="correlationTab">
       <attribute name="title">
        <string>Correlation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="PlotView" name="correlationView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>samples</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="iterationNumber">
        <property name="font">
         <font>
          <family>Abyssinica SIL</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acquireButton">
        <property name="text">
         <string>Acquire</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="recordButton">
        <property name="text">
         <string>Record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mkCorrButton">
        <property name="text">
         <string>mkCorr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Temp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="temperatureSpinBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>164</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotView</class>
   <extends>QWidget</extends>
   <header>QatPlotWidgets/PlotView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
